custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -125,7 +127,7 @@ class ArgumentsOptions:
     CLASS_LABELS = "--class-labels"
     CLASS_LABELS_FILE = "--class-labels-file"
 
-    MAIN_COMMAND = "drum"
+    MAIN_COMMAND = "drum" if not DEBUG else "./custom_model_runner/bin/drum"
```

Comment:
I'm not sure I love this, doesn't it assume directory?

Reply:
it assumes the working directory is the repo, which is fairly common for DR repos. I did this because I was tired of having to either re-install a local copy of drum or run the tests in docker. I can remove it if you'd prefer